content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,181 @@
+
+## Services
+
+yax.im updated their servers from prosody 0.11 to [prosody-trunk](https://hg.prosody.im/trunk/file/d0e6d5bc7ea2/CHANGES). Read more about the changes [here](https://yaxim.org/blog/2020/04/23/new-prosody-on-yax-dot-im/). Unfortunately with a first [outage](https://yaxim.org/blog/2020/04/27/multi-day-message-archive-outage/) :(
+
+On April 25th, [observe.jabber.network](https://observe.jabber.network) was launched into public beta. It offers free monitoring-as-a-service for federated XMPP domains, checking connectivity via c2s and s2s, as well as optionally in-band registration and federated XMPP pings. Application details and more information are available on the website.
+
+
+## Extensions and specifications
+
+
+### Deferred
+
+### Updated
+
+- Version 1.18.0 of XEP-0060 (Publish-Subscribe) has been released.
+Abstract:
+This specification defines an XMPP protocol extension for generic publish-subscribe functionality. The protocol enables XMPP entities to create nodes (topics) at a pubsub service and publish information at those nodes; an event notification (with or without payload) is then broadcasted to all entities that have subscribed to the node. Pubsub 
+therefore adheres to the classic Observer design pattern and can serve as the foundation for a wide variety of applications, including news feeds, content syndication, rich presence, geolocation, workflow systems, network management systems, and any other application that requires event notifications.
+Changelog: 
+* Properly specifiy that an empty  is invalid on publish. (jsc)
+URL: https://xmpp.org/extensions/xep-0060.html
+
+- Version 0.4 of XEP-0333 (Chat Markers) has been released.
+Abstract:
+This specification describes a solution of marking the last received, displayed and acknowledged message in a chat.
+Changelog:
+* Add notes about usage within MUCs. (mw)
+URL: https://xmpp.org/extensions/xep-0333.html
+
+- Version 0.3.0 of XEP-0389 (Extensible In-Band Registration) has been
+released.
+Abstract:
+This specification defines an XMPP protocol extension for in-band registration with instant messaging servers and other services with which an XMPP entity may initiate a stream. It aims to improve upon the state of the art and replace XEP-0077: In-Band Registration by allowing multi-factor registration mechanisms, and account recovery.
+Changelog:
+* Move back to experimental.
+* Add IQ variant of registration/recovery.
+* Allow client to select between multiple flows. (ssw)
+URL: https://xmpp.org/extensions/xep-0389.html
+
+Note: The information in the XEP list at https://xmpp.org/extensions/
+is updated by a separate automated process and may be stale at the
+time this email is sent. The XEP documents linked herein are up-to-
+date.
```

Comment:
I think this note can be removed.

Reply:
Hmm, its always in the previous one and is also information in the emails. Are you sure?
Other opinions @jcbrand @Ppjet6 ?